@xen-orchestra/lite/src/components/pool/dashboard/cpuUsage/PoolCpuUsageChart.vue

```diff
@@ -50,51 +45,39 @@ const timestampStartHostStatsComputed = computed(
   () => hostLastWeekStats.timestampStart?.value
 );
 
-const timestampStartWmStatsComputed = computed(
-  () => vmLastWeekStats.timestampStart?.value
-);
-
-const labelSerie = computed(() => t("stacked-cpu-usage"));
-
 const data = computed<LinearChartData>(() => {
   const result = new Map<string, { date: number; value: number }>();
 
-  const setResult = (
-    stats: HostStats | VmStats | undefined,
-    timestampStart = 0,
-    index = 0
-  ) => {
-    const avgCpuUsage = getAvgCpuUsage(stats?.cpus, {
-      nSequence: customMaxValue.value / 100,
-    });
-
-    if (avgCpuUsage === undefined) {
-      return;
+  const addResult = (stats: HostStats) => {
+    const cpus = Object.values(stats.cpus);
+    for (let hourIndex = 0; hourIndex < cpus[0].length; hourIndex++) {
+      const timestamp =
+        ((timestampStartHostStatsComputed.value ?? 0) + hourIndex * 3600) *
```

Comment:
To avoid hard coded value.
```suggestion
        ((timestampStartHostStatsComputed.value ?? 0) + hourIndex * RRD_STEP_FROM_STRING.hours) *
```